src/openzeppelin/token/erc20/presets/ERC20.cairo

```diff
@@ -1,149 +1,105 @@
-    }() -> (symbol: felt):
-    let (symbol) = ERC20.symbol()
-    return (symbol)
-end
+func symbol{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (symbol: felt) {
+    return ERC20.symbol();
+}
 
 @view
-func totalSupply{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }() -> (totalSupply: Uint256):
-    let (totalSupply: Uint256) = ERC20.total_supply()
-    return (totalSupply)
-end
+func totalSupply{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
+    totalSupply: Uint256
+) {
+    let (totalSupply: Uint256) = ERC20.total_supply();
+    return (totalSupply,);
+}
 
 @view
-func decimals{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }() -> (decimals: felt):
-    let (decimals) = ERC20.decimals()
-    return (decimals)
-end
+func decimals{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
+    decimals: felt
+) {
+    return ERC20.decimals();
+}
 
 @view
-func balanceOf{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(account: felt) -> (balance: Uint256):
-    let (balance: Uint256) = ERC20.balance_of(account)
-    return (balance)
-end
+func balanceOf{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(account: felt) -> (
+    balance: Uint256
+) {
+    return ERC20.balance_of(account);
+}
 
 @view
-func allowance{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(owner: felt, spender: felt) -> (remaining: Uint256):
-    let (remaining: Uint256) = ERC20.allowance(owner, spender)
-    return (remaining)
-end
-
-#
-# Externals
-#
+func allowance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    owner: felt, spender: felt
+) -> (remaining: Uint256) {
+    return ERC20.allowance(owner, spender);
+}
+
+//
+// Externals
+//
 
 @external
-func transfer{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(recipient: felt, amount: Uint256) -> (success: felt):
-    ERC20.transfer(recipient, amount)
-    return (TRUE)
-end
+func transfer{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    recipient: felt, amount: Uint256
+) -> (success: felt) {
+    let (success) = ERC20.transfer(recipient, amount);
+    return (success,);
+}
 
 @external
-func transferFrom{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(
-        sender: felt,
-        recipient: felt,
-        amount: Uint256
-    ) -> (success: felt):
-    ERC20.transfer_from(sender, recipient, amount)
-    return (TRUE)
-end
+func transferFrom{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    sender: felt, recipient: felt, amount: Uint256
+) -> (success: felt) {
+    let (success) = ERC20.transfer_from(sender, recipient, amount);
+    return (success,);
```

Comment:
we're on the right way. all of these externals should be re-exportable as they're so let's follow the above functions